lib/resolvers/os_release_resolver.rb

```diff
@@ -61,6 +62,10 @@ def process_name
                                 @fact_list[:name].split(' ')[0].strip
                               end
         end
+
+        def normalize_opensuse_identifier
+          @fact_list[:identifier] = 'opensuse' if @fact_list[:identifier] =~ /opensuse/
```

Comment:
``/opensuse/i`` as the regex to not be case sensitive

Reply:
fixed